src/blocks/expandable/frontend.scss

```diff
@@ -12,11 +12,7 @@
 
 	&::after {
 		bottom: 0;
-		background: linear-gradient(
-			180deg,
-			transparent 0%,
-			var(--expandable-fade-color, #fff) 80%
-		);
```

Comment:
Ah well, I actually liked the old style, pity the linter doesn't 😆

Reply:
Same :/